libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts

```diff
@@ -4,9 +4,12 @@ import { CommonModule } from '@angular/common';
 import { ConsultancyRoutingModule } from './consultancy.routing';
 
 import { ConsultancyPageComponent } from './main/consultancy-page/consultancy-page.component';
+import { ConsultancyHeroSectionComponent } from './components/consultancy-hero-section/consultancy-hero-section.component';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
 
```

Comment:
Kindly, observe the order of imports; the elements imports should appear before local imports.